src/tests/Microsoft.PowerFx.Core.Tests/ExpressionTestCases/Remove.txt

```diff
@@ -133,4 +131,8 @@ Table({Field1:2,Field2:"moon",Field3:DateTime(2022,2,1,0,0,0,0),Field4:false},{F
 Errors: Error 7-10: Name isn't valid. 'Foo' isn't recognized.|Error 12-22: The specified column 'Field1' does not exist.|Error 0-30: The function 'Remove' has some invalid arguments.
    
 >> Remove(Foo, Bar)
-Errors: Error 7-10: Name isn't valid. 'Foo' isn't recognized.|Error 12-15: Name isn't valid. 'Bar' isn't recognized.|Error 0-16: The function 'Remove' has some invalid arguments.
\ No newline at end of file
+Errors: Error 7-10: Name isn't valid. 'Foo' isn't recognized.|Error 12-15: Name isn't valid. 'Bar' isn't recognized.|Error 0-16: The function 'Remove' has some invalid arguments.
+
+// Remove propagates error.
+>> Remove(t1, {Field1:2,Field2:"moon",Field3:DateTime(2022,2,1,0,0,0,0),Field4:false})
```

Comment:
>moon [](http://example.com/codeflow?start=32&length=4)

Can you change this to something more explicit, such as `Field2:"not in the table"` to make it clear why the error is being returned?